apps/ai_reports/serializers.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+
+from apps.ai_reports.models import AiReport
+
+
+class AiReportSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AiReport
+        fields = (
+            "explanation",
+            "category",
+            "comment",
+        )
```

Comment:
@goapunk I added this serializer to make use of nested serializers for the comment serializer, but didn't work, so I used the one-to-one relation `comment.ai_report` in the comment serializer in the end. Shall I keep this serializer and the viewset of AiReport for future api use cases?